(dp1
S'output'
p2
S"<type 'exceptions.TypeError'> 'NoneType' object has no attribute '__getitem__'"
p3
sS'layer'
p4
S'/home/www-data/web2py/applications/reliefList/controllers/default.py'
p5
sS'code'
p6
S'<code object <module> at 0x7f77cd19c4b0, file "/home/www-data/web2py/applications/reliefList/controllers/default.py", line 2>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/www-data/web2py/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/home/www-data/web2py/applications/reliefList/controllers/default.py", line 231, in <module>\n  File "/home/www-data/web2py/gluon/globals.py", line 414, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/www-data/web2py/applications/reliefList/controllers/default.py", line 186, in paymentDetails\n    prodId = session.productIdStripe[\'id\']\nTypeError: \'NoneType\' object has no attribute \'__getitem__\'\n'
p11
s.